gopherage/cmd/junit/junit.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package junit
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/tools/cover"
+
+	"k8s.io/test-infra/gopherage/pkg/cov/junit"
+)
+
+type flags struct {
```

Comment:
the struct is not exported. But, the fields are.

Reply:
To be consistent with the behavior here: https://github.com/kubernetes/test-infra/blob/master/gopherage/cmd/filter/filter.go

Reply:
You can export the var without exporting the struct if you need to use the values. But, if you are not using them outside this pkg, they should not be exported